NMS-14959: Document Time Series tags (#5772)
OpenNMS/opennms: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

. Obtain an activation key for the Cloud Services Connector.
. Enable cloud services connectivity.

=== Update Cloud Services Connector files

> The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the {page-component-title}-hosted cloud service.

Reviewer: ```suggestion
The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the OpenNMS-hosted cloud service.
```
This is referring to the Appliance Portal, but at the time of writing marketing suggested we not use the "Appliance Service" term. Technically it is The OpenNMS Group hosting the cloud service, but OpenNMS works in this context.

The packages for this extension are installed by default in Horizon 31 and later, but do need to be configured and activated.
You can use `yum` or `apt` to make sure the `opennms-cloud-plugin` package is up to date with the most current version of the plugin.

To ensure that the feature is activated on service restarts, add the feature to a file in `featuresBoot.d`:

[source, console]
echo "opennms-plugin-cloud-core wait-for-kar=opennms-plugin-cloud" | sudo tee ${OPENNMS_HOME}/etc/featuresBoot.d/plugin-cloud.boot

=== Configure Cloud Services

You will need to add settings to tell your {page-component-title} instance to load the plugin on service start and to direct storage to the cloud service.
If you are running Horizon 31 or later, copy the example `timeseries.properties` file to your working directory:

[source, console]
cp ${OPENNMS_HOME}/etc/examples/opennms.properties.d/hosted_tsdb.timeseries.properties ${OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties

> The following tags can be configured in `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties`:

Reviewer: ```suggestion
You can configure the following tags in `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties`:
```

Reviewer 2: ```suggestion
You can configure custom tags in the following contexts in `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties`:
```

* Asset
* Interface
* Node
* Service

> * Resources

Reviewer 2: ```suggestion
* Resource
```

> * Custom

Reviewer 2: I *think* `custom` refers to metadata, though I haven't tested that.

Reviewer 2: Just went back to the source to verify
```suggestion
* Metadata
```

Below are examples of tag definitions:

[source, properties]
----

> org.opennms.timeseries.tin.metatags.tag.node=${node:label}

Reviewer: should we use the tags listed in the bulleted list as part of the example tag definitions?

Author: I think having these should communicate the concept, but I can update this to match the list if it would make more sense. :)

Reviewer: No, that's ok. Thanks!
